modules/userman/email.go

```diff
@@ -132,3 +146,18 @@ func (m *Module) HandleEmailSignUp() http.HandlerFunc {
 		json.NewEncoder(w).Encode(map[string]interface{}{"user": req, "token": token})
 	}
 }
+
+func hashPassword(pwd string) string {
+	//Generates a new hash from the given password
+	hash, err := bcrypt.GenerateFromPassword([]byte(pwd), bcrypt.MinCost)
+	if err != nil {
+		panic(err.Error)
```

Comment:
Can we return an InternalServerError instead of panic?

Reply:
Sure, but I would suggest doing both so that the administrator can see in the logs that an error was raised.